client/post-editor/editor-gutenberg-opt-in-sidebar/index.jsx

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Button from 'components/button';
+import EditorGutenbergOptInDialog from 'post-editor/editor-gutenberg-opt-in-dialog';
+
+function EditorGutenbergOptInSidebar( { translate } ) {
+	return (
+		<div className="editor-gutenberg-opt-in-sidebar">
+			<img src="/calypso/images/illustrations/gutenberg-mini.svg" alt="" />
+			<p>{ translate( 'Try our new editor and level-up your layout.' ) }</p>
+			<Button onClick={ EditorGutenbergOptInDialog.onShowDialog } action="show">
```

Comment:
Let's decouple this a bit from the EditorGutenbergOptInDialog, and pass through an onClick handler. The parent caller can deal with that.

```jsx 
function EditorGutenbergOptInSidebar( { translate, onClick } ) {
//...
<Button onClick={ onClick } action="show">
```

It's also okay to rename the pass through action handler here, to something more descriptive like `displayGutenbergOptIn`, but `onClick` is fine if there's only one handler expected.